gapis/gfxapi/gfxapi.proto

```diff
@@ -50,6 +50,9 @@ enum ResourceType {
 	ShaderResource = 5;
 	// ProgramResource represents the Program resource type
 	ProgramResource = 6;
+	// UninitializedTextureResource represents texture resource type
+	// which has been generated but does not have dimensions yet.
+	UninitializedTextureResource = 7;
```

Comment:
Add a TODO to address https://github.com/google/gapid/issues/237, and I'll approve.